src/inputs/Switch/Switch.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './Switch.module.css'
+import { h } from 'preact'
+
+interface Props {
+	checked: boolean
+	disabled?: boolean
+	id?: string
+	onChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+const Switch: React.FC<Props> = ({ checked = false, disabled = false, id, onChange }) => {
+	return (
+		<label className='switch'>
+			<input type='checkbox' checked={checked} disabled={disabled} id={id} onChange={onChange} />
```

Comment:
**suggestion:** You can use `hidden` attribute on this input to hide it, and style it more easy.